packages/server/src/routes/members/createMember.ts

```diff
@@ -411,8 +421,59 @@ export const createMember = (
       }
     });
 
+    _addEmailToMailingLists(
+      sgClient,
+      call.body.emailAddress,
+      call.body.subscribeToNewsletter
+    );
+
     return {
       body: newOpsData,
       status: 201
     };
   });
+
+async function _addEmailToMailingLists(
+  sgClient: SgClient,
+  emailAddress: string,
+  subscribeToNewsletter?: boolean
+) {
+  // Don't fail CREATE_MEMBER due to SendGrid API.
+  try {
+    const response = await sgClient.request({
+      method: "POST",
+      url: "/v3/contactdb/recipients",
+      body: [{ email: emailAddress }]
+    });
+
+    const persisted_recipients = response[1].persisted_recipients;
+    if (persisted_recipients.length === 0) {
+      throw new Error(`SendGrid could not add recipient`);
+    }
+    const recipient_id = persisted_recipients[0];
+
+    const APP_REGISTRATION = "5460644";
```

Comment:
should this be part of config?

Reply:
What would be the benefit?

Reply:
I did it but just wondering what the benefit is if we're not splitting out test list vs prod list

Reply:
Maybe at some point we will lol. But i think generally since we've been putting credentials/ids for all the other apps in config it's a nice thing to do